docs/run/estimate-job-run-time.mdx

```diff
@@ -54,7 +56,7 @@ Output:
 ```
 ## View the estimated job usage on IBM Quantum Platform
 
-You can view the estimated usage (in quantum seconds) in two places on IBM Quantum Platform:
+You can view the estimated usage (in seconds) in two places on IBM Quantum Platform:
```

Comment:
It's not at all obvious that "job usage" is the same as the quantum time that was mentioned above.

Reply:
Good point - I changed it to this "You can view the estimated usage (how much quantum time the job will take to run) in two places on IBM Quantum Platform"